pkg/ebpf/tracee.go

```diff
@@ -609,11 +616,16 @@ func (t *Tracee) initDerivationTable() error {
 	pathResolver := containers.InitPathResolver(&t.pidsInMntns)
 	soLoader := sharedobjs.InitContainersSymbolsLoader(&pathResolver, 1024)
 
-	symbolsLoadedFilters := t.config.Filter.ArgFilter.GetEventFilters(events.SymbolsLoaded)
+	symbolsLoadedFilters := map[string]filters.Filter{}
+	for filterScope := range t.config.FilterScopes.UserSpaceMap() {
+		for k, v := range filterScope.ArgFilter.GetEventFilters(events.SymbolsLoaded) {
+			symbolsLoadedFilters[k] = v
```

Comment:
As we have already reached go 1.18, this can be replaced by `maps.Copy(symbolsLoadedFilters, filterScope.ArgFilter.GetEventFilters(events.SymbolsLoaded))`.